ui/src/components/Absentee/Absentee.vue

```diff
@@ -38,7 +38,7 @@ export default {
     this.$amplitude
       .getInstance()
       .logEvent('Page Visit', { page: 'Journey Page', component: 'Absentee' });
-  }
+  },
```

Comment:
When I add this change locally and then run ci-check ...
> localhost:~/projects/Five-Fifths-Voter/ui$ yarn ci-check   
> yarn run v1.22.11
> $ yarn format:diff
> $ prettier --list-different "src/**/*.{js,vue,scss}"
> src/components/Absentee/Absentee.vue
> error Command failed with exit code 1.
> info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
> error Command failed with exit code 1.
> info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

If I run prettier against the file it removes the trailing comma.

Somehow your prettier is using
```js
"trailingComma": "es5",
```
but it's not in the package.json. Is it possible that you have a prettier config file that is adding this?
https://prettier.io/docs/en/configuration.html

Reply:
I think that `"trailingComma": "es5",` is the default behaviour for prettier: https://prettier.io/docs/en/options.html#trailing-commas

Reply:
Also, before running `yarn ci-check` make sure to have updated all yarn dependencies in the branch with `yarn install` or simply `yarn`

Reply:
You are exactly right. I was taking a shortcut and not running yarn from your PR. The `trailingComma` did indeed change in prettier 2.0.